guide/admin/grouppermissions.md

```diff
@@ -0,0 +1,31 @@
+## User Handbook
+### Assign permissions to User Groups
+
+To assign a permission set to a User Group this requires you to be in the Administrator Control Panel, you can do this by clicking your username to the top right of your screen, and then clicking **Administration**. The example below will show you how this is done:
+
+![Administration Link Screenshot](http://i.imgur.com/WZ1Zjx9.gif)
+
+Once you are in the Administration Panel, you'll see some navigation tabs flowing down the left hand side, click on **Permissions** to reveal the configuration page to the right hand side.
+
+![Permissions Screenshot](http://i.imgur.com/yX4ETf0.png)
+
+On the configuration page, you'll see a number of different permissions that can be applied to user groups, both pre-created on the initial install and to user groups that you create, as shown below:
+
+![SS Group Permissions](http://i.imgur.com/rTPP7Ii.png)
+
+In this guide, we're going to use the usergroup that we created in [Create/Modify/Delete](usergroups.md) called **Staff Member** and apply the appropriate permissions to that group.
+
+### Assigning permissions to a User Group
+
+To start off with it is essential that you know what permissions you wish to assign to the user group you selected, in this example, as it's a staff member user group, we're going to be assigning all of the available permissions, you can do that by hovering your mouse after the icon of each of the available permissions, there you'll find a small drop-down icon, click that and select the user group you wish to attach this permission to, as you can see below:
+
+![Anim - Assigning Permissions](http://i.imgur.com/NwxeTLI.gif)
```

Comment:
What does "Anim" mean? Animation? Please use a long description, for accessibility.